packages/experience/src/pages/Consent/ScopeGroup/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { type Nullable } from '@silverhand/essentials';
+import classNames from 'classnames';
+import { useState, useCallback } from 'react';
+
+import DownArrowIcon from '@/assets/icons/arrow-down.svg';
+import CheckMark from '@/assets/icons/check-mark.svg';
+import { onKeyDownHandler } from '@/utils/a11y';
+
+import * as styles from './index.module.scss';
+
+type ScopeGroupProps = {
+  readonly groupName: string;
+  readonly isAutoExpand?: boolean;
+  readonly scopes: Array<{
+    id: string;
+    name: string;
+    description?: Nullable<string>; // Organization scope description cloud be `null`
+  }>;
+};
+
+const ScopeGroup = ({ groupName, scopes, isAutoExpand = false }: ScopeGroupProps) => {
+  const [expanded, setExpanded] = useState(isAutoExpand);
+
```

Comment:
Should we add a `useEffect` hook to set expanded status? Otherwise, if the `isAutoExpanded` is changed in the parent component, this child component will not reflect the change.

Reply:
The `isAutoExpanded` will not change, since related data won't be modified, only for display purpose.